doc/.wordlist.txt

```diff
@@ -27,6 +27,7 @@ Btrfs
 bugfix
 bugfixes
 Centos
```

Comment:
Can you remove "Centos" then?
Not sure why it's in there - it's only used in a code block afaics.